jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -280,64 +283,78 @@ def test_wcslink_offset(self):
 
         # Check the coordinates display: Last loaded is on top.
         # Within bounds of non-reference image but out of bounds of reference image.
-        self.viewer.on_mouse_or_key_event({'event': 'mousemove', 'domain': {'x': 10, 'y': 3}})
-        assert self.viewer.label_mouseover.pixel in ('x=07.0 y=00.0', 'x=07.0 y=-0.0')
-        assert self.viewer.label_mouseover.value == '+0.00000e+00 '
-        assert self.viewer.label_mouseover.world_ra_deg == '3.5817877198'
-        assert self.viewer.label_mouseover.world_dec_deg == '-30.3919358920'
-        assert self.viewer.label_mouseover.unreliable_world
-        assert self.viewer.label_mouseover.unreliable_pixel
+        label_mouseover = self.imviz.app.session.application._tools['g-coords-info']
+        label_mouseover._viewer_mouse_event(self.viewer,
+                                            {'event': 'mousemove', 'domain': {'x': 10, 'y': 3}})
+        assert label_mouseover.as_text() in [('Pixel x=07.0 y=00.0 Value +0.00000e+00',
+                                              'World 00h14m19.6291s -30d23m30.9692s (ICRS)',
+                                              '3.5817877198 -30.3919358920 (deg)'),
+                                             ('Pixel x=07.0 y=-0.0 Value +0.00000e+00',
+                                              'World 00h14m19.6291s -30d23m30.9692s (ICRS)',
+                                              '3.5817877198 -30.3919358920 (deg)')]

```

Comment:
This makes it clearer that only first row is non-deterministic.
```suggestion
        lmtext = label_mouseover.as_text()
        assert lmtext[0] in ('Pixel x=07.0 y=00.0 Value +0.00000e+00', 'Pixel x=07.0 y=-0.0 Value +0.00000e+00')
        assert lmtext[1:] == ('World 00h14m19.6291s -30d23m30.9692s (ICRS)',
                              '3.5817877198 -30.3919358920 (deg)')
```